index.js

```diff
@@ -1,11 +1,17 @@
 'use strict'
 
 const path = require('path');
-const symbols = require('./symbols')
+const symbols = require('./symbols');
 
 const include_dir = path.resolve(__dirname, 'include');
+const defRoot = path.resolve(__dirname, 'def')
+const def = {
+    js_native_api_def: path.join(defRoot, 'js_native_api.def'),
+    node_api_def: path.join(defRoot, 'node_api.def')
+}
 
 module.exports = {
     include_dir,
+    def,
```

Comment:
Might be worth suffixing with `_dir`, similar to `include_dir`.

```suggestion
    def_dir,
```